docs/modules/ROOT/pages/discover/slsa-conformity.adoc

```diff
@@ -1 +1,150 @@
 = Supply chain security through SLSA conformity
+
+== Supply-chain Levels for Software Artifacts (SLSA)
+
+Supply-chain Levels for Software Artifacts (SLSA) is a security framework produced through industry collaboration. We use this framework as a guide for reinforcing the build process we use for your applications, to better secure your software supply chain.
+
+SLSA assigns two primary responsibilities to build platforms like {ProductName}:
```

Comment:
```suggestion
The SLSA Build (v1.0) specification assigns two primary responsibilities to build platforms like {ProductName}:
```
The specification may change, so we can specifically indicate which track/version we are referencing: https://slsa.dev/spec-stages#versioning

This is also consistent with the additional links at the bottom.